frontend/pages/DashboardPage/sections/MetricsHostCounts/_styles.scss

```diff
@@ -1,68 +1,30 @@
 .metrics-host-counts {
   display: grid;
   gap: $pad-smedium;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(3, auto);
 
-  // >= 320px and < 576px
-  // Each card is on its own row, up to 3 rows.
-  // 12pt gap
-  @media (min-width: $break-mobile-xs) {
-    grid-template-columns: 1fr;
-    grid-template-rows: repeat(3, auto);
-  }
-
-  // >= 576px and < 768px
-  @media (min-width: $break-mobile-md) {
+  // >= 768px
+  @media (min-width: $break-sm) {
     display: grid;
 
-    /** Default layout for 3 cards 
-    The first card (TotalHosts) is on a row by itself.
-    The second and third cards are on the second row. */
-    grid-template-columns: repeat(2, 1fr);
-    grid-template-rows: auto auto;
-
-    & > *:first-child {
-      grid-column: 1 / -1;
-    }
-
-    /* Adjust layout for only 2 cards to be on the same row */
-    &:has(> :nth-child(2):last-child) {
-      grid-template-columns: repeat(2, 1fr);
-      grid-template-rows: auto;
-
-      & > *:first-child,
-      & > *:nth-child(2) {
-        grid-column: auto;
-      }
-    }
-
-    /* Adjust layout for only 1 card to take a single row */
-    &:has(> :first-child:last-child) {
-      grid-template-columns: 1fr;
-      grid-template-rows: auto;
-    }
-  }
-
-  // >= 768px
-  // Up to 3 cards are in a single row.
-  // 16pt gap
```

Comment:
This skips a transitive layout that is visually very minor but introduces a fair amount of maintenance overhead.

Reply:
On re-review this is arguable. Maybe the comments make it seem more complex.